### PR TITLE
Fix #45: Added support for Bootstrap checkbox/radio toggle buttons

### LINKED ISSUE #45: Add support of Bootstrap checkbox/radio toggle buttons to ActiveField
Migrated from https://github.com/yiisoft/yii2/issues/7519

Add support of [these button groups](http://getbootstrap.com/javascript/#buttons-checkbox-radio) to bootstrap/ActiveField.

### PR DESCRIPTION
**Pull request by klimov-paul**
Fix #45: Added support for Bootstrap checkbox/radio toggle buttons

### CODE REVIEW

**Comment by SilverFire** [comment]
Will it be merged?

File: ToggleButtonGroup.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    /**
+     * @inheritdoc
+     */
+    public function run()
+    {
+        if (!isset($this->options['item'])) {
+            $this->options['item'] = [$this, 'renderItem'];
+        }
+        switch ($this->type) {
+            case 'checkbox' :
```
Comment (SilverFire) [inline ToggleButtonGroup.php]: redundant space before ` :`

**Comment by SilverFire** [comment]
Looks good as for me

**Comment by klimov-paul** [comment]
Merged

**Pull request merged**